pyam/utils.py

```diff
@@ -129,6 +129,18 @@ def format_data(df):
         missing = list(set(IAMC_IDX) - set(df.columns))
         raise ValueError("missing required columns `{}`!".format(missing))
 
+    if 'notes' in df.columns:
+        logger().info('Ignoring notes column in dataframe')
+        df.drop(columns='notes',inplace=True)
+        # remove SSP Public Database row
+        database_label_rows = [
+            'SSP Public Database' in model
+            for model in df.model
+        ]
+        df.drop(df[database_label_rows].index, inplace=True)
```

Comment:
I think this can be a one-liner:

```
df = df[~df.model.str.contains('SSP Public Database')]
```

Reply:
From what I remember of the database version, this could also be:

```
df = df.iloc[:-2]
```

Just a thought!

Reply:
I'd argue against blindly removing the last column - we may want to add more columns in later snapshot-export versions (e.g., user/timestamp of upload).

The cleanest way would be to remove all columns that are not in the `IAMC_IDX` or years (i.e., int).

Reply:
`df = df.iloc[:-2]` removes just the last row right? I'm not sure how best to implement removing any columns not in `IAMC_IDX` so will leave that up to you to add if you want.